src/Couchbase.Lite.Shared/API/Error/CouchbaseLiteErrorMessage.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Couchbase.Lite
+{
+    internal static class CouchbaseLiteErrorMessage
+    {
+        //Database error messages
+        internal const string CreateDBDirectoryFailed = "Unable to create database directory";
+        //Database - Copy
+        internal const string ResolveDefaultDirectoryFailed = "Failed to resolve a default directory! If you have overriden the IDefaultDirectoryResolver interface, please check it.  Otherwise please file a bug report.";
+        internal const string InvalidPath = "Path.Combine failed to return a non-null value!";
```

Comment:
Path.Combine?

Reply:
This is a .NET specific exception that I put in to silence a ReSharper warning.  It doesn't need to be in this list.